src/components/FirebaseAuth.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+import firebase from "firebase"
+import StyleFirebaseAuth from "react-firebaseui/StyledFirebaseAuth"
+
+firebase.initializeApp({
+   apiKey: "AIzaSyAXDqsWK4quNVaf9-YV2e28NsxkfA9rzJA",
+   authDomain: "dreamerscholars.firebaseapp.com"
+})
+
+class FirebaseAuth extends Component {
+   state = { isSignedIn: false }
+   uiConfig = {
+      signInFlow: "popup",
+      signInOptions: [
+         firebase.auth.GoogleAuthProvider.PROVIDER_ID,
+         firebase.auth.EmailAuthProvider.PROVIDER_ID,
+         firebase.auth.FacebookAuthProvider.PROVIDER_ID
+      ],
+      callbacks: {
+         signInSuccess: () => false
```

Comment:
[signInWithSuccess](https://github.com/firebase/firebaseui-web#signinsuccesscurrentuser-credential-redirecturl) is being deprecated so I think this works:
```suggestion
         signInSuccessWithAuthResult: () => false
```